charts/datadog-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: datadog-operator
-version: 2.10.0-dev
+version: 2.10.1-dev
```

Comment:
```suggestion
version: 2.10.0-dev.1
```
We can't bump from `2.10.1-dev` to `2.10.0`.

Reply:
Thanks! I wasn't sure how to handle the version bump